pulp_smash/pulp3/constants.py

```diff
@@ -22,13 +22,9 @@
 
 DISTRIBUTION_PATH = urljoin(BASE_PATH, "distributions/")
 
-DOWNLOAD_POLICIES = ("immediate", "on_demand", "streamed")
+DOWNLOAD_POLICIES = ("immediate", )
 
-JWT_PATH = urljoin(BASE_PATH, "jwt/")
```

Comment:
:1st_place_medal:

Reply:
We kept this one around since  JWT could be added back based on the discussions at the time.

Reply:
Since then though we removed the JWT entirely so I think we should remove this too.